src/main/kotlin/com/terraformation/backend/nursery/db/NurseryBatchEventData.kt

```diff
@@ -0,0 +1,11 @@
+package com.terraformation.backend.nursery.db
+
+import com.terraformation.backend.db.default_schema.SpeciesId
+import com.terraformation.backend.db.nursery.BatchId
+
+data class NurseryBatchEventData(
```

Comment:
Can we directly use `NurserySeedlingBatchReadyEvent` instead of this? It doesn't seem like this class will ever be used for anything except constructing one of those events (that idea is even encoded into this class's name) so may as well skip the intermediate step.

Reply:
Seems like a reasonable change!